plugin/src/main/groovy/com/novoda/gradle/command/Install.groovy

```diff
@@ -23,4 +23,12 @@ class Install extends AdbTask {
         customFlags ?: ''
     }
 
+    @Override
+    protected handleCommandOutput(def text) {
+        super.handleCommandOutput(text)
+        def matcher = text =~ /Failure \[(.*?)\]/
```

Comment:
I know with Nougat release, Android team did a lot of refactoring in adb. I remember I listened in this [podcast](http://androidbackstage.blogspot.de/2016/07/episode-53-adb-on-adb.html). They know forward exit code and error stream of the command being run on the device when we run `adb shell`. 

Unfortunately `adb install` always succeeds with this message in it. I think I need to open a bug report about this.